modules/ROOT/pages/single-sign-on-with-kerberos.adoc

```diff
@@ -289,13 +289,13 @@ If you want Bonita engine to create the accounts on the fly when a user accessin
 NOTE: Activating this option means any user logged in on the domain trying to access Bonita will have an account created automatically in Bonita Database.
 
 . If your Domain Controller is correctly configured, you are done. +
-Then you can start the bundle and try to access a portal page, an app page or a form URL (or just `http://<host>:<port>/bonita[?tenant=<tenantId>]`) and make sure that you are automatically logged in.
+Then you can start the bundle and try to access any Bonita Application page, an app page or a form URL (or just `http://<host>:<port>/bonita[?tenant=<tenantId>]`) and make sure that you are automatically logged in.
 
 Note that if you try to access `http://<bundle host>:<port>/bonita/login.jsp`, then you won't be redirected as this page still needs to be accessible in order for the tenant administrator (or another user if you set the property `kerberos.auth.standard.allowed` to true or define a whitelist with the property `auth.tenant.standard.whitelist`) to be able to log in without an account on AD.
 
 == Logout behavior
 
-The most commonly used solution is to hide the logout button from the portal. Users are logged in as long as they don't close their web browser (unless their session times out). +
+The most commonly used solution is to hide the logout button from the Bonita Layout. Users are logged in as long as they don't close their web browser (unless their session times out). +
```

Comment:
Users are logged in as long as they don't close their web browser (unless their session times out). => Users are logged in as long as they don't close their web browser and the session does not time out.

Reply:
Done, I modified other phrases like this as well